src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTag.cs

```diff
@@ -254,5 +257,19 @@ private async Task StartToolTipServiceAsync(IToolTipPresenter toolTipPresenter)
 
             toolTipPresenter.StartOrUpdate(_textView.TextSnapshot.CreateTrackingSpan(_span.Start, _span.Length, SpanTrackingMode.EdgeInclusive), uiList);
         }
+
+        private void Adornment_MouseLeftButtonDown(object sender, MouseButtonEventArgs e)
```

Comment:
can you address this @akhera99 ?

Reply:
I thought since we discussed that there wasn't a need for the disconnects, then this was not necessary? What benefit is there to separate out the UI related code?

Reply:
This is more likely to be a clean-up suggestion if we don't need to worry about event disconnects.

It seems like in the `CreateElement()` method above is creating a stack panel and putting UI elements in it. I wonder if it possible to put all these into a xaml file?